example_analysis/aggregate_4/tile-based/aggregate_4_smk_test.ipynb

```diff
@@ -0,0 +1,1987 @@
+{
```

Comment:
<p>For my own understanding, why do these two numbers not add up to the 50,000 rows that are loaded in above?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cheeseman-lab/OpticalPooledScreens/pull/14/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>Because clean_cell_data removes any cells that do not have an assigned perturbation. The filter_single_gene option is whether to subset to data that only has a singular assigned perturbation. We can also select the more likely perturbation (which happens when set to false)</p>